minicore/src/arch/x86.rs

```diff
@@ -0,0 +1,285 @@
+use crate::abstractions::{bit::*, bitvec::*};
+
+pub(crate) mod upstream {
+    #[cfg(target_arch = "x86")]
+    pub use core::arch::x86::*;
+    #[cfg(target_arch = "x86_64")]
+    pub use core::arch::x86_64::*;
+}
+
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
```

Comment:
Shouldn't this be guarding the entire file?

Reply:
Ah, no, actually, the specs should not be guarded: the specs are for modeling the x86 architecture, but there is no reason to disable them for other archs.